benchmarks/n_root.out

```diff
@@ -0,0 +1 @@
+1.5157178699512095
```

Comment:
I'm guessing that the tests are failing because of the missing newline here? Trying `turnt --save` could help.